86153.9 <c-29#257> Created logger
86153.9 <c-29#257> appl_initialize was called
86153.9 <c-29#257:3> Logger was updated
86153.9 <c-29#257:3> Started intializing cube lib
86153.9 <c-29#257:3> Finished intializing cube lib with worker, starting to work
86154 <c-29#257:3> requestCubes => [61] Buffer size: 0
86154 <c-29#257:3> Sent requestCubes signal to root
86154 <c-29#257:3> Received 1 cubes from root
86154 <c-29#257:3> The main loop continues
86154 <c-29#257:3> Started solving a cube
86154 <c-29#257:3> Cube failed
86154 <c-29#257:3> Added failed cube
86154 <c-29#257:3> returnFailedCubes => [61] Buffer size: 3
86154 <c-29#257:3> Sent 1 failed cubes to root
86154 <c-29#257:3> Received receivedFailedCubes signal from root
86154 <c-29#257:3> requestCubes => [61] Buffer size: 0
86154 <c-29#257:3> Sent requestCubes signal to root
86154 <c-29#257:3> Received 1 cubes from root
86154 <c-29#257:3> The main loop continues
86154 <c-29#257:3> Started solving a cube
86210 <c-29#257:3> appl_interrupt was called
86210 <c-29#257:3> Interrupting worker
86210 <c-29#257:3> appl_interrupt was called
86210 <c-29#257:3> appl_withdraw was called
86210 <c-29#257:3> Started withdraw thread
86210 <c-29#257:3> STOP (0.00s since last cb)
86210 <c-29#257:3> Solving interrupted
86210 <c-29#257:3> The main loop continues
86210 <c-29#257:3> Leaving the main loop due to an interruption
86210 <c-29#257:3> Finished withdrawing lib
86210 <c-29#257:3> Enter destructor of CubeWorker
86210 <c-29#257:3> Time waiting for messages: 0.000
86210 <c-29#257:3> Enter destructor of CubeWorkerInterface
86210 <c-29#257:3> Enter destructor of CubeRoot
86210 <c-29#257:3> Finished withdraw thread
86272.1 <c-29#257:3> Enter destructor
86272.1 <c-29#257:3> Time working: 56.086
86272.1 <c-29#257:3> Time suspended: 0.000
86272.1 <c-29#257:3> Joined cleanup thread
86272.1 <c-29#257:3> Exit destructor
86272.1 <c-29#257:3> Destructing logger
